### DISCUSSION TITLE
How to turn off tooltips in taskbar and menu items?

### DISCUSSION THREAD

**Discussion by developer1105** [lxqt/lxqt#2568] [Q&A] [answered]
Is this option available in config files?

**Comment by stefonarch** [comment]
You mean in lxqt-panel? The answer is no, but I think you could edit your theme and make them invisible. Using kvantum you can set a  delay for 9999ms  for showing them but this will have overall effect.

**Comment by tsujan** [comment]
Tooltips are controlled by Qt and, to some extent, by the active widget style.

**Comment by developer1105** [comment]
It could be like Xfce, in dark themes the tooltips are also dark (white letters over black background). 
The theme I am using now has light tooltips, black letters over yellow background, very distracting for a black theme.

**Reply by tsujan** [reply]
> It could be like Xfce

It isn't about XFCE or LXQt but about the style of the toolkit: GTK in XFCE and Qt in LXQt.

**Reply by Gnomasz** [reply]
Just to be sure: is it one of the standard LXQt themes?

**Reply by tsujan** [reply]
If a Panel tooltip isn't styled by a CSS (=LXQt theme), it'll be drawn by the active Qt widget style (Fusion, Kvantum, Breeze,...). If it's styled by a CSS completely, the Qt widget style will be overridden. That's true for all Panel widgets.

**Comment by stefonarch** [comment, marked as answer]
> The theme I am using now has light tooltips, black letters over yellow background, very distracting for a black theme.

As said, you could edit the theme, copy it to `~/.local/share/lxqt/themes` and edit in `lxqt-panel.qss` the section (mostly at the beginning):

```
QToolTip {
    border: 1px solid #2b2b29;
    background: #363532;
    padding: 2px;
    margin: 0px;
    color: #f2f1f0;
}
```
Panel needs reloading after saving.

**Comment by developer1105** [comment]
Changing the color solved it. Thanks.
